PlanningPoker/Repostitories/UserRepository.cs

```diff
@@ -0,0 +1,70 @@
+﻿using PlanningPoker.Context;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using PlanningPoker.Models;
+
+namespace PlanningPoker.Repostitories
+{
+    public class UserRepository : IUserRepository
+    {
+        private readonly PokerContext _context;
+
+        public UserRepository(PokerContext context)
+        {
+            _context = context;
+        }
+        public User AddAsync(User entity)
+        {
+            _context.Users.Add(entity);
+            return entity;
+        }
+
+        public void DeleteAsync(string name)
+        {
+            var entity = _context.Users.FirstOrDefault(x => x.Name == name);
+            if (entity != null) _context.Users.Remove(entity);
+        }
+
+        public IEnumerable<User> GetUsersByRoomId(string id)
+        {
+            var users = _context.Users.Where(x => x.RoomId == id);
```

Comment:
You are returning IQueryable here, I would suggest to change IEnumerable<User> to IReadOnlyCollection

Reply:
Also you should add AsNoTracking()